### PR TITLE
fix: jwt-auth add consumer_schema to the _M

### PR DESCRIPTION
**Pull request by Jaycean**
Signed-off-by: Jaycean <jinchen_jacean@163.com>

### What this PR does / why we need it:
fix: #2553 
- jwt-auth add consumer_schema to the _M

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Review by spacewander — APPROVED** [review]

**Comment by liuxiran** [comment]
add a test case: get jwt-auth schema would be better @Jaycean  :)

File: t/plugin/jwt-auth.t [hunk 1]
```diff
@@ -519,3 +519,23 @@ GET /t
...
+--- request
+GET /apisix/admin/schema/plugins/jwt-auth?schema_type=consumer
+--- response_body
+{"required":["key"],"properties":{"exp":{"minimum":1,"type":"integer"},"algorithm":{"type":"string","default":"HS256","enum":["HS256","HS512","RS256"]},"base64_secret":{"default":false,"type":"boolean"},"secret":{"type":"string"},"key":{"type":"string"}},"additionalProperties":false,"type":"object"}
+--- no_error_log
+[error]
+
+
+
+=== TEST 26: get the schema by error schema_type
```
Comment (spacewander) [inline t/plugin/jwt-auth.t:535]: The name of test case should be `by the default schema_type`.

**Review by spacewander — CHANGES_REQUESTED** [review]

File: t/plugin/jwt-auth.t (hunk 1 above)
Comment (Jaycean) [reply t/plugin/jwt-auth.t:535]: ok, here is the test case when passing schema_type to obtain consumer_schema, I added the test case obtained by default schema_type

**Review by membphis — APPROVED** [review]

**Review by Jaycean — COMMENTED** [review]
> add a test case: get jwt-auth schema would be better @Jaycean :)

done

**Review by liuxiran — APPROVED** [review]
LGTM

File: t/plugin/jwt-auth.t (hunk 1 above)
Comment (Jaycean) [reply t/plugin/jwt-auth.t:535]: @spacewander done.

**Review by spacewander — APPROVED** [review]

**Pull request merged**